test/src/examples_complete_test.go

```diff
@@ -6,76 +6,31 @@ import (
 	"testing"
 
 	"github.com/gruntwork-io/terratest/modules/terraform"
-	"github.com/stretchr/testify/assert"
 )
 
 // Test the Terraform module in examples/complete using Terratest.
 func TestExamplesComplete(t *testing.T) {
 	t.Parallel()
 
-	randId := strconv.Itoa(rand.Intn(100000))
-	attributes := []string{randId}
-
-	exampleInput := "Hello, world!"
+	// We always include a random attribute so that parallel tests and AWS resources do not interfere with each
+	// other
+	randID := strconv.Itoa(rand.Intn(100000))
+	attributes := []string{randID}
 
 	terraformOptions := &terraform.Options{
 		// The path to where our Terraform code is located
 		TerraformDir: "../../examples/complete",
 		Upgrade:      true,
 		// Variables to pass to our Terraform code using -var-file options
 		VarFiles: []string{"fixtures.us-east-2.tfvars"},
-		// We always include a random attribute so that parallel tests
-		// and AWS resources do not interfere with each other
 		Vars: map[string]interface{}{
-			"attributes": attributes,
-			"example":    exampleInput,
+			"attributes":       attributes,
+			"enable_cis_1_2_0": true,
 		},
 	}
 	// At the end of the test, run `terraform destroy` to clean up any resources that were created
 	defer terraform.Destroy(t, terraformOptions)
 
 	// This will run `terraform init` and `terraform apply` and fail the test if there are any errors
 	terraform.InitAndApply(t, terraformOptions)
-
```

Comment:
add tests for the TF outputs.
If the IDs are completely random, we can use `assert.NotEmpty` to at least check if we have any outputs